test.js

```diff
@@ -900,8 +872,7 @@ test('createAsyncIterator creates AsyncIterator from another AsyncIterator', ()
 })
 
 test('createAsyncIterator creates AsyncIterator for string literal', () => {
-  // $FlowFixMe expected function override to apply.
-  var iterator = createAsyncIterator('ABC')
+  var iterator = createAsyncIterator((('ABC': any): {length: number}))
```

Comment:
In Flow the primitive types such as strings are not objects. However, `{length: number}` is treated like an object. Perhaps an interface instead should work, but so far I couldn't get it to work.